networkdb/networkdb.pb.go

```diff
@@ -476,7 +476,7 @@ func (m *CompoundMessage) GetMessages() []*CompoundMessage_SimpleMessage {
 type CompoundMessage_SimpleMessage struct {
 	// Bytestring payload of a message constructed using
 	// other message type definitions.
-	Payload []byte `protobuf:"bytes,1,opt,name=Payload,json=payload,proto3" json:"Payload,omitempty"`
```

Comment:
Wonder what caused this change. Was it because of a protobuf bump? If so, I wonder why CI didn't fail before

Reply:
since the base image changed, Im guessing this line got executed
```
RUN apt-get update && apt-get -y install iptables \
                 protobuf-compiler
```

Reply:
oooh. we install from the distro. we should consider installing it ourself and pick a fixed version; similar to what's done in swarmkit? https://github.com/docker/swarmkit/blob/01966d400f4fb9ec573c5fd17181e8340f25ae85/.circleci/config.yml#L16

https://github.com/docker/swarmkit/blob/01966d400f4fb9ec573c5fd17181e8340f25ae85/.circleci/config.yml#L62-L68